cnf-certification-test/accesscontrol/securitycontextcontainer/securitycontextcontainer.go

```diff
@@ -163,7 +163,9 @@ const (
 	CategoryID4String       = "CategoryID4(anything not matching lower category)"
 )
 
-// print the strings
+// String is find the string of Category.
```

Comment:
// String is find the string of Category.
-> 
// String converts the category to a string